tmt-app/src/Styling/CustomStyling.js

```diff
@@ -197,3 +209,19 @@ export const FlexContainer = styled(Container)({
   alignItems: "center",
   justifyContent: "center",
 });
+
+export const DetailCard = styled(Card)({
+  backgroundColor: "#F3E8DE",
+  borderRadius: "36px",
+  padding: "2.25rem",
+  boxShadow: "none",
+  [breakpoints.up("xs")]: {
+    margin: "0.75rem",
+  },
+  [breakpoints.up("sm")]: {
+    margin: "1.25rem",
+  },
+  [breakpoints.up("md")]: {
+    margin: "1.75rem",
+  },
+});
```

Comment:
Hey, take a read through this [pull request](https://github.com/BobaTalks/internship-tracker/pull/124/files/ca08c57ce93827efaa5a5d5ec779f0c1c48748b0..bd377b018ec4e4577c0300a42133e3274f584b8d#diff-abb3c003f564b18a7f0cf35bdbd535ae69165b4493a6ffc6868f7e8b0d4959c9)

I think both teams here are coming across some common points in larger picture styling.

The sentiment to make as few CSS changes as possible contributes to the desire to minimize technical/design debt on our side in the future.

Can we do a quick check to see what common styling on the TMT side can be identified and adjust the base mui components at the theme level instead of custom/styled components like this or use of "custom" variants that are essentially the same as having blocks of custom css?

Reply:
Sounds good! I will make the changes. Just for future reference, what would we do if we need to use a mui component that has been overridden?

Reply:
If it's more specific than the most common use case, then we can just add in the props for styling when we use it! We just want to take the most common design elements, our _theme_ and apply those changes at the most generic level